frontend/pendingTranslations.ftl

```diff
@@ -91,3 +91,14 @@ profile-free-onboarding--addon-item-headline-1 = They work anywhere you need an
 profile-free-onboarding--addon-item-description-1 = Anywhere you need to enter an email, you can use a { -brand-name-relay } email mask. You can even generate new masks directly from your { -brand-name-firefox } password manager. 
 profile-free-onboarding--addon-item-headline-2 = The { -brand-name-relay } extension makes it easy
 profile-free-onboarding--addon-item-description-2 = Whenever you see an email address field, the { -brand-name-firefox-relay } browser extension will autofill any email mask you want to use. No need to remember each mask!
+
+# 2023 Holiday promo
+# Variables:
+# $couponCode (string) - the coupon code.
+# $couponExpiryDate (string) - the coupon expiry date, for example, Dec 31, 2023.
```

Comment:
I think you need to put these on the line before the strings that actually use those variables, otherwise localisers won't see them at the right time.

Tip: [check out Pontoon](https://pontoon.mozilla.org/es-MX/firefox-relay-website/), then you'll have an idea of how the localisers work. They don't edit FTL files directly :)